README.md

```diff
@@ -5,9 +5,26 @@ Prezto — Instantly Awesome Zsh
 
 Prezto is the configuration framework for [Zsh][1]; it enriches the command line
 interface environment with sane defaults, aliases, functions, auto completion,
-and prompt themes. 
+and prompt themes.
 
-This repository is being actively maintained by a community of users. 
+Upgrading from sorin-ionescu/prezto
+-----------------------------------
+
+Prezto's [original maintainer][original] abandoned the project. This is a
+young, community-maintained fork, which was created [in December 2016][fork].
```

Comment:
I used link references because in my experience inline links make paragraphs hard to read. Readability of the source document is one of the [design principles of Markdown](https://daringfireball.net/projects/markdown/syntax#philosophy), and striving for that makes sense to me. Long lines containing links continually interrupt the flow.